### ISSUE TITLE
FileNotFoundException trying to read scoverage-data

### ISSUE THREAD

**Issue by zachgk** [scoverage/scoverage-maven-plugin#72]
I am running 1.4.0-M5 for an aggregating multi-module project (https://github.com/apache/incubator-mxnet/tree/master/scala-package). It is throwing the following error on the scoverage.report.CoverageAggregator: 
```
[ERROR] /home/ubuntu/incubator-mxnet/scala-package/init/target/scoverage-data (Is a directory)
java.io.FileNotFoundException: /home/ubuntu/incubator-mxnet/scala-package/init/target/scoverage-data (Is a directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at scala.xml.Source$.fromFile(XML.scala:19)
	at scala.xml.factory.XMLLoader.loadFile(XMLLoader.scala:48)
	at scala.xml.factory.XMLLoader.loadFile$(XMLLoader.scala:48)
	at scala.xml.XML$.loadFile(XML.scala:58)
	at scoverage.report.ScoverageXmlReader$.read(ScoverageXmlReader.scala:15)
	at scoverage.report.CoverageAggregator$.$anonfun$aggregatedCoverage$1(CoverageAggregator.scala:29)
	at scoverage.report.CoverageAggregator$.$anonfun$aggregatedCoverage$1$adapted(CoverageAggregator.scala:28)
	at scala.collection.Iterator.foreach(Iterator.scala:937)
	at scala.collection.Iterator.foreach$(Iterator.scala:937)
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1425)
	at scala.collection.IterableLike.foreach(IterableLike.scala:70)
	at scala.collection.IterableLike.foreach$(IterableLike.scala:69)
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54)
	at scoverage.report.CoverageAggregator$.aggregatedCoverage(CoverageAggregator.scala:28)
  </organization>
	at scoverage.report.CoverageAggregator.aggregatedCoverage(CoverageAggregator.scala)
	at org.scoverage.plugin.SCoverageReportMojo.generateAggregatedReports(SCoverageReportMojo.java:511)
	at org.scoverage.plugin.SCoverageReportMojo.canGenerateReport(SCoverageReportMojo.java:287)
	at org.scoverage.plugin.SCoverageReportMojo.execute(SCoverageReportMojo.java:373)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:207)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:153)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:145)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:116)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:80)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:307)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:193)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:106)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:863)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:288)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:199)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[ERROR]
```

When I looked into this, it seems like scoverage-maven-plugin depends on the "aggregate raw coverage data" PR on scalac-coverage-plugin(https://github.com/scoverage/scalac-scoverage-plugin/pull/241) that did not get put into 1.4.0-M5. Everything runs fine on the sonatype snapshot version. Is there an ETA until either the full release of 1.4 or another milestone version?

**Comment by gslowikowski** [comment]
Hi @zachgk 

My mistake, sorry. The next version could be `-RC1` supporting Scala `2.13.0-RC1`, but there are problems with this version of Scala which are blockers for `scalac-scoverage-plugin` (see scoverage/scalac-scoverage-plugin#250).

Version `1.4.0` will be released when Scala `2.13.0` will be available.

**Comment by gslowikowski** [comment]
Version `1.4.0-RC1` was released. Please upgrade.
